docker-compose.ytt.yaml

```diff
@@ -135,6 +135,29 @@ services:
     volumes:
       - db-data:/var/lib/postgresql/data
       - ./postgres/init:/docker-entrypoint-initdb.d/
+  
+  virtuoso:
+    image: tenforce/virtuoso:latest
```

Comment:
Ideally, do not se the latest tag but a specific version. This way we can make sure that the things we rely on do not change.

Reply:
[virtuoso7.2.5](https://hub.docker.com/layers/tenforce/virtuoso/virtuoso7.2.5/images/sha256-de97286328aa0babb9e06e9626321d91363ba3f260529b9083d1fa02f36ad664?context=explore) seems to be the latest tag.